addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/HiddenFilesScanRule.java

```diff
@@ -322,6 +323,42 @@ private String readPayloadsFile() {
         return "";
     }
 
+    private boolean doesNotMatch(String responseBody, List<String> notContentStrings) {
+        boolean matches = true;
+        if (!notContentStrings.isEmpty()) {
+            for (String notContentStr : notContentStrings) {
+                // True if the content does indeed NOT match
+                matches = !responseBody.contains(notContentStr);
+                if (!matches) {
+                    return matches;
+                }
+            }
+        }
+        return matches;
+    }
+
+    private boolean doesMatch(String responseBody, List<String> contentStrings) {
+        boolean matches = true;
```

Comment:
This could be removed and return the boolean values instead (as we know which one where).